gnomad/utils/vcf.py

```diff
@@ -796,6 +875,85 @@ def make_info_dict(
             },
         }
         info_dict.update(popmax_dict)
+    elif grpmax:
+        grpmax_dict = {
+            f"{prefix}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Genetic ancestry group with the maximum allele"
+                    f" frequency{description_text}"
+                ),
+            },
+            f"{prefix}AC{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Allele count in the genetic ancestry group with the maximum allele"
+                    f" frequency{description_text}"
+                ),
+            },
+            f"{prefix}AN{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Total number of alleles in the genetic ancestry group with the"
+                    f" maximum allele frequency{description_text}"
+                ),
+            },
+            f"{prefix}AF{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Maximum allele frequency across genetic ancestry"
+                    f" groups{description_text}"
+                ),
+            },
+            f"{prefix}nhomalt{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Count of homozygous individuals in the genetic ancestry group"
+                    f" with the maximum allele frequency{description_text}"
+                ),
+            },
+            f"{prefix}faf95{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Filtering allele frequency (using Poisson 95% CI) for the genetic"
+                    " ancestry group with the maximum allele"
+                    f" frequency{description_text}"
+                ),
+            },
+        }
+        info_dict.update(grpmax_dict)
+    elif fafmax:
```

Comment:
Should it be `if fafmax:`? I thought `popmax` is degraded, that's why we're copying what it has to `grpmax`, but `grpmax` and `fafmax` coexist, no?

Reply:
So this is a weird conditional structure I tried to update for v4 but I'm going to revisit this. It is built this way because we call each metric one by one but that is kinda weird. I'll update it again